test/server/cards/01_SOR/events/Resupply.spec.ts

```diff
@@ -0,0 +1,33 @@
+describe('Resupply', function() {
+    integration(function(contextRef) {
+        describe('Resupply\'s ability', function() {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['resupply'],
+                        leader: 'leia-organa#alliance-general'
+                    },
+                    player2: {
+                    }
```

Comment:
Nit (non-blocking): Do we need to define player2 here? I believe there are defaults